app/addons/documents/index-editor/components.react.jsx

```diff
@@ -326,10 +351,12 @@ function (app, FauxtonAPI, React, Stores, Actions, Components, ReactComponents)
         return;
       }
 
+      var doc = this.refs.designDocSelector.getDesignDoc();
```

Comment:
hmmm, not feeling comfortable by calling a getter on a view to get data. we used to do that with backbone a lot and it got us really into trouble.

maybe we can put that into a store and let the component fire an action? the dumb component does not need to be connected directly to the store, it could get the data updates as props from the parent component